src/index.js

```diff
@@ -1,47 +1,100 @@
 import './index.css';
 import Icon from './more.png';
 import Refresh from './refresh.png';
+import Edit from './edit.png';
+import Remove from './remove.png';
+import Add from './plus.png';
+import Todo from './Todo.js';
 
+const tasksArr = JSON.parse(localStorage.getItem('todos'));
+const todoAddInput = document.querySelector('#todo-add-input');
+const refresImg = document.querySelector('#refresh');
 const todoListUl = document.querySelector('.todo-list');
-const todosArr = [
-  {
-    description: 'Wash the dishes',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Read a book',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Make my bed',
-    completed: false,
-    index: 2,
-  },
-];
+const btnAddTodo = document.querySelector('#btn-add-todo');
 
 const populateTodo = () => {
-  let todoItems = ` <li class="todo-item todo-title">
-            <h2> Today's To Do</h2>
-            <img src=${Refresh} alt="refresh" class="todo-hold">
-    </li>
-    <li class="todo-item todo-add">
-        <input type="text" placeholder="Add to your list...">
-    </li>`;
-  todosArr.forEach((todo) => {
-    todoItems = `${todoItems} <li class="todo-item todos">
-        <div class="todo-wrap">
-            <input type="checkbox">
-            <p class="todo-text"> ${todo.description}</p>
-        </div>
-        <img src="${Icon}" alt="move" class="todo-hold">
-        </li>`;
-  });
+  let todoItems = '';
+  refresImg.src = Refresh;
+  btnAddTodo.src = Add;
+  const storageTasks = JSON.parse(localStorage.getItem('todos'));
```

Comment:
- [ ] if there is no key in local storage with the name `todos` in the local storage, you will get this error
![image](https://user-images.githubusercontent.com/24703171/200388806-20c4d136-f7ed-4938-a83b-0e61d252c75a.png)
because `taskArr` will contain null. Kindly fix this error.
#### Hint: you can fix it by checking for null before you use the `populateTodo` method.